SampleConnectorWebJob/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Sample.Connector
+{
+    using System;
+    using System.Threading.Tasks;
+    using System.Diagnostics;
+    using System.Net.Http;
+    using Microsoft.WindowsAzure.Storage.Table;
+    using Microsoft.WindowsAzure.Storage.Queue;
+
+    public class Program
+    {
+        private static HttpClient httpClient = new HttpClient();
+        private static AzureStorageQueueProvider queueProvider;
+        private static AzureTableProvider azureTableProvider;
+        private static CloudTable pageJobMappingTable;
+        private static TimeSpan sleepTime = TimeSpan.FromSeconds(Convert.ToInt32(Settings.SleepTimeInSec));
+        private static TimeSpan queueVisibilityTimeOutInSec = TimeSpan.FromSeconds(Convert.ToInt32(Settings.QueueVisibilityTimeOutInSec));
+
+        public static void Main()
```

Comment:
why not WebJob SDK ?

Reply:
Added existing console application project as a web job project, 
Web Job SDK can also be used, with the option of add new project as a web job project.
Going with this in current push. Improvements can be there in future.